### PR TITLE
Codeowners: count all Markdown files as documentation

### PR DESCRIPTION
**Pull request by RDIL**
## PR Summary
Makes users that are Codeowners of `Documentation` own all markdown files
  

## PR Context  
Popular files like the README have no codeowners.  

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.  
- **User-facing changes**
    - [X] Not Applicable
    - **OR**  
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [X] N/A or can only be tested interactively
    - **OR**
    - [X] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [X] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 # Area: Documentation
 .github/    @joeyaiello @TravisEz13
+*.md        @joeyaiello @TravisEz13
```
[the remark this answers is not in the document]
Comment (TravisEz13) [reply .github/CODEOWNERS]: I think we would have to split up the markdown files... maybe by folder.

[the remark this answers is not in the document]
Comment (RDIL) [reply .github/CODEOWNERS]: Sounds like it would work.  If you want, I could also do it (but its fine if not)

**Comment by RDIL** [comment]
Should I close it?

**Comment by iSazonov** [comment]
@RDIL TravisEz13 is self-assigned so please wait.

**Comment by TravisEz13** [comment]
Let's go ahead and close this.

**Comment by RDIL** [comment]
Alright.

**Pull request closed without merging**